README.md

```diff
@@ -1,13 +1,15 @@
 # TUF
 
+This repository is **not** in active maintenance. Please see the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) file to learn about overall Notary Project.
+
 TUF is a project to implement the full TUF specification in a registry native way. This may
 require upstream TUF spec changes or extensions, as there are some differences between the
 registry model and common usage to other TUF use cases. This project will use existing
 registry extensions where available but may need its own document types in addition.
 
-The initial version of Notary ran as an additional service on a registry, so was not
+The initial version of notary TUF-based implementation ran as an additional service on a registry, so was not
```

Comment:
```suggestion
The notary TUF-based implementation ran as an additional service on a registry, so was not
```

Removing the phrase "initial version" will help clarify that there isn't any newer version of notary TUF-based implementation.